src/renderer/marketplace/incorporation/checkout/checkout-container.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { kycOperations } from 'common/kyc';
+import { incorporationsSelectors } from 'common/incorporations';
+import { withStyles } from '@material-ui/core/styles';
+import { getIncorporationPrice } from '../common';
+import { ethGasStationInfoOperations } from 'common/eth-gas-station';
+import PaymentCheckout from '../../common/payment-checkout';
+import * as CheckoutUtil from '../../common/checkout-util';
```

Comment:
this functionality available in the helper, should use marketplace common component instead, right?

Reply:
Yes, i thought I did this refactor in the last changes, maybe it got lost when solving merge conflicts, my bad. I've created a new commit with the changes. Now, incorporations checkout will use the marketplace commons components and CheckoutUtil.js got removed from the wallet